### PR TITLE
docs: Adjust Non-UI APIs docs for better searchability

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?

Some of the docs inside non-UI APIs don't follow the feature template and they have a worse searchability with the new search. Also, some of the tables for showing the support status are doesn't match the overall style and I noticed that the Android TV docs page is missing from the toc.yml.

### <samp>🤖 Generated by Copilot at ada6d8c</samp>

This pull request updates various articles in the Uno Platform documentation that cover Uno-specific information for various UWP features. It adds tip sections with links to the UWP documentation, and it replaces emoji with plain text symbols in some support tables. The purpose is to provide more information, guidance, and consistency for the readers.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):
[Message from Dominik and Sasha on Teams
](https://teams.microsoft.com/l/message/19:787ad1de1af24252b9041bcbd4ec1ca3@thread.tacv2/1699643883904?tenantId=a297d6c0-b635-41a3-b1e3-558efe71e413&groupId=f55cdec8-036b-4efa-8e1a-c4129ba9bf12&parentMessageId=1698880353319&teamName=Uno%20Platform&channelName=Docs%20and%20Getting%20Started&createdTime=1699643883904)

### CODE REVIEW

File: doc/articles/features/windows-ui-xaml-window.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ uid: Uno.Features.WinUIWindow
 
 # Windows.UI.Xaml.Window
```
Comment (MartinZikmund) [inline doc/articles/features/windows-ui-xaml-window.md:5]: Just say `Window` here and update to use `Microsoft.UI.Xaml.Window`, as WUX is more or less obsolete

File: doc/articles/features/windows-ui-xaml-application.md [hunk 2]
```diff
@@ -2,7 +2,12 @@
 uid: Uno.Features.WinUIApplication
 ---
 
-# Windows.UI.Xaml.Application
+# Application
+
+> [!TIP]
+> This article covers Uno-specific information for `Windows.UI.Xaml.Application`. For a full description of the feature and instructions on using it, consult the UWP documentation: https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.application
```
Comment (MartinZikmund) [inline doc/articles/features/windows-ui-xaml-application.md:8]: Use Microsoft.UI.Xaml.Application here

File: doc/articles/features/Lottie.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ uid: Uno.Features.Lottie
 
 # Lottie for Uno
 
+> [!TIP]
+> This article covers Uno-specific information for `Lottie`. For a full description of the feature and instructions on using it, consult the UWP documentation: https://learn.microsoft.com/en-us/windows/communitytoolkit/animations/lottie
+
+* The `Microsoft.Toolkit.Uwp.UI.Lottie` namespace provides classes for rendering Lottie animations in a `Windows.UI.Xaml.Controls.AnimatedVisualPlayer`.
```
Comment (MartinZikmund) [inline doc/articles/features/Lottie.md:10]: And use `Microsoft.UI.Xaml` here

**Review by MartinZikmund — APPROVED** [review]

File: doc/articles/features/windows-devices-geolocation.md [hunk 4]
```diff
@@ -2,7 +2,12 @@
 uid: Uno.Features.WDGeolocation
 ---
 
-# Uno Support for Windows.Devices.Geolocation
+# Geolocation
+
+> [!TIP]
+> This article covers Uno-specific information for Geolocation. For a full description of the feature and instructions on using it, consult the UWP documentation: https://learn.microsoft.com/en-us/uwp/api/windows.devices.geolocation
+
+* The `Windows.Devices.Geolocation.Geolocator` provides APIs for getting the current location or tracking the device's location over time. Location information may come from estimating a position from beacons like Wi-Fi access points and cell towers, from the device's IP address, or it may come from other sources such as a GNSS or GPS device.
```
Comment (agneszitte) [inline doc/articles/features/windows-devices-geolocation.md:10]:
```suggestion
* The `Windows.Devices.Geolocation.Geolocator` provides APIs for getting the current location or tracking the device's location over time. Location information may come from estimating a position from beacons like Wi-Fi access points and cell towers, from the device's IP address, or may come from other sources such as a GNSS or GPS device.
```

File: doc/articles/features/windows-system-display.md [hunk 5]
```diff
@@ -2,7 +2,12 @@
 uid: Uno.Features.WSDisplay
 ---
 
-# Uno Support for Windows.System.Display APIs
+# Windows.System.Display APIs
+
+> [!TIP]
+> This article covers Uno-specific information for `Windows.System.Display` namespace. For a full description of the feature and instructions on using it, consult the UWP documentation: https://learn.microsoft.com/en-us/uwp/api/windows.system.display
+
+* The `Windows.System.Display.DisplayRequest` class enables an application to request for keeping the device's screen on.
```
Comment (agneszitte) [inline doc/articles/features/windows-system-display.md:10]:
```suggestion
> This article covers Uno-specific information for `Windows.System.Display` namespace. For a full description of the feature and instructions on using it, consult the UWP documentation: https://learn.microsoft.com/en-us/uwp/api/windows.system.display

* The `Windows.System.Display.DisplayRequest` class enables an application to request to keep the device's screen on.
```

File: doc/articles/features/windows-devices-geolocation.md (hunk 4 above)
Comment (morning4coffe-dev) [reply doc/articles/features/windows-devices-geolocation.md:10]: Thanks for the suggestions, Agnès! 😇

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

**Pull request merged**